assignment2/assignment2/Presentation/FriendList/FriendListViewController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  FriendListViewController.swift
+//  assignment2
+//
+//  Created by 이승헌 on 2022/10/12.
+//
+
+import UIKit
+
+let deviceWidth: CGFloat = UIScreen.main.bounds.width
+let deviceHeight: CGFloat = UIScreen.main.bounds.height
```

Comment:
만약, 이렇게 전역적으로 관리해야하는 상수가 있다면 따로 metric 열거형 파일을 만들어서 관리해보는 것은 어떤가요? 

예를 들어서 Metric.swift 파일을 하나 만들고 아래와 같이 관리하는 것이죠!!
```suggestion
// MARK: - Metric Enum

public enum Metric {
    
}

// MARK: - Common

extension Metric {
    static let deviceWidth: CGFloat = UIScreen.main.bounds.width
    static let deviceHeight: CGFloat = UIScreen.main.bounds.height
}
```

그리고 필요한 곳에서 타입 프로퍼티로 접근하면 되겠죠?
Metric.deviceWidth .. 이렇게 사용하면 됩니다!!